frontend/components/puzzle.jsx

```diff
@@ -10,17 +26,22 @@ export default class Puzzle extends React.Component {
     });
 
     this.state = {
-      correct: null
+      correct: null,
+      reset: this.resetCorrect.bind(this)
```

Comment:
dont think you need to keep track of this in state, later, you can refer to it as `this.resetCorrect`